CommandLine/Accept.cs

```diff
@@ -86,6 +86,29 @@ public static class Accept
                 return null;
             }));
 
+        public static ArgumentsRule LegalFilePathsOnly(
```

Comment:
@livarcocc  I think this makes sense but making it consistent will require a couple of small breaking changes. 

@polarapfel  Order matters in that the first rule to indicate an error will short circuit the rest of the rules. In terms of compilation, order doesn't matter as these extensions are accepting and returning the same type.

So one idea to make these more consistent might be to use "With" as the prefix only for extension methods in this class, and leaving it off when they're not extension methods, yielding, e.g.:

```
Accept.OneOrMoreArguments()
           .WithLegalFilePathsOnly()
           .WithSuggestionsFrom(_ => ... ); // assuming we want to get clever
```

The breaking changes in order to make this consistent would be:

```
Accept.SuggestionsFrom("x", "y"); // formerly Accept.WithSuggestionsFrom(...). The extension method overloads Accept.WithSuggestionsFrom(this ArgumentsRule rule) would be unchanged.

Accept.ExactlyOneArgument()
           .WithExistingFilesOnly(); // Formerly "ExistingFilesOnly". 
```
In the last example, "With" is kind of awkward, which is why I didn't use it. I then followed this pattern for `WithLegalFilePathsOnly`. Consistency and English grammer... Choose one.